test/services/CalculationServiceSpec.scala

```diff
@@ -555,6 +555,79 @@ class CalculationServiceSpec extends UnitSpec {
           result.upperTaxRate shouldEqual Some(28)
         }
       }
+
+      //############ Flat Rate PRR Tests ########################
+      "return £0 tax owed for a Disposal Date of 06-10-2016, Acquisition Date of 05-04-2015, Days Eligible of 5, Gain of £2000 and " +
+        "PRR of £2015" in {
+        val result = CalculationService.calculateCapitalGainsTax(
+          calculationType = "flat",
+          customerType = "individual",
+          priorDisposal = "No",
+          annualExemptAmount = Some(0),
+          otherPropertiesAmt = Some(0),
+          isVulnerable = Some("No"),
+          currentIncome = Some(0),
+          personalAllowanceAmt = Some(0),
+          disposalValue = 2000,
+          disposalCosts = 0,
+          acquisitionValueAmt = 0,
+          acquisitionCostsAmt = 0,
+          revaluedAmount = 0,
+          revaluationCost = 0,
+          improvementsAmt = 0,
+          reliefs = 0,
+          allowableLossesAmt = 0,
+          entReliefClaimed = "No",
+          acquisitionDate = Some("2015-04-05"),
+          disposalDate = Some("2016-10-06"),
+          isClaimingPRR = Some("Yes"),
+          daysClaimed = Some(5),
+          daysClaimedAfter = Some(0)
+        )
+        result.totalGain shouldEqual 2000
+        result.taxOwed shouldEqual 0
+      }
+
+      "return £9498.56 tax owed for a Disposal Date of 03-10-2016, Acquisition Date of 20-04-2013, Days Eligible of 0, Gain of £32000 and " +
+        "PRR of £43,548" should {
+        val result = CalculationService.calculateCapitalGainsTax(
+          calculationType = "flat",
+          customerType = "individual",
+          priorDisposal = "No",
+          annualExemptAmount = Some(0),
+          otherPropertiesAmt = Some(0),
+          isVulnerable = Some("No"),
+          currentIncome = Some(0),
+          personalAllowanceAmt = Some(0),
+          disposalValue = 100000,
+          disposalCosts = 0,
+          acquisitionValueAmt = 0,
+          acquisitionCostsAmt = 0,
+          revaluedAmount = 0,
+          revaluationCost = 0,
+          improvementsAmt = 0,
+          reliefs = 0,
+          allowableLossesAmt = 0,
+          entReliefClaimed = "No",
+          acquisitionDate = Some("2013-04-20"),
+          disposalDate = Some("2016-10-03"),
+          isClaimingPRR = Some("Yes"),
+          daysClaimed = Some(0),
+          daysClaimedAfter = Some(0)
+        )
+
+        "have total gain of £100,000" in {
+          result.totalGain shouldEqual 100000
+        }
+
+        "have base tax gain of £32000" in {
+          result.baseTaxGain shouldEqual 32000
+        }
+
+        "have total taxed owed of £9498.56" in {
+          result.taxOwed shouldEqual 9498.56
+        }
+      }
     }
```

Comment:
Can we add an additional scenario here that has values accepted for everything, and that are in pence. To ensure that all the rounding is executed accurately and returns the right expected result. 

You could use one of the existing tests and modify it for adding in PRR